### ISSUE TITLE
phpfn interwiki links broken

### ISSUE THREAD

**Issue by selfthinker** [splitbrain/dokuwiki#1402]
php.net seems to have changed their short URLs. E.g. http://php.net/preg-replace would previously redirect to http://php.net/manual/en/function.preg-replace.php (or even another country-dependent version). And our e.g. `[[phpfn>preg-replace]]` interwiki links point to the former.

In my quick search I couldn't find if this was deliberate by PHP.net or a bug on their system. As this feature appears to be missing since a couple of months now, it's probably best to change our interwiki link anyway.

**Comment by splitbrain** [comment]
That link works fine for me?

**Comment by selfthinker** [comment]
Hmm, strange, it does work for me now as well. I'm 100% sure it didn't work for me 20 minutes ago. And I also remember having tried it a couple of weeks ago when it didn't work either.

I just noticed there is a mistake in my description. phpfn links to https, so e.g. https://php.net/preg-replace redirects to https://secure.php.net/preg-replace ... not sure if that might be a reason for flakiness?

What prompted me to look into this was this change: https://www.dokuwiki.org/plugin:batchedit?rev=1442158994&do=diff
And I've noticed other users do the same on other pages...
